subsys/caf/modules/buttons.c

```diff
@@ -485,25 +515,52 @@ static void button_pressed_fn(struct k_work *work)
 	}
 }
 
+static int gpio_validate(void)
+{
+	for (size_t i = 0; i < ARRAY_SIZE(col); i++) {
+		ssize_t idx = get_gpio_idx(col[i].port);
+
+		if (idx < 0) {
+			return -EINVAL;
+		}
+		if (!gpio_devs[idx].has_gpiote) {
+			return -ENOTSUP;
```

Comment:
`cols` are used to set signal (GPIO outputs). Do they need to support GPIOTE too ?